package.json

```diff
@@ -43,7 +43,7 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "axios": "^0.19.0"
+    "axios": "next"
```

Comment:
I'd definitely prefer this value to match the value in `devDependencies`

Reply:
I'd like to reproduce that 401(unauthorized) error but now it seems that the API endpoint work as expected. Now  I could use the client to fetch the data using headers.😅
```javascript
require("dotenv").config();
const { WakaTimeClient, RANGE } = require("wakatime-client");

const {
  WAKATIME_API_KEY: wakatimeApiKey
} = process.env;

const wakatime = new WakaTimeClient(wakatimeApiKey);

async function main() {
  const stats = await wakatime.getMyStats({ range: RANGE.LAST_7_DAYS });
  console.log(stats);
}

main();

```

But it did occurred a week ago even I used a refreshed API key.  This maybe an issue from `wakatime`'s server as I mentioned in #95 and now the issued has been resolved🤔.

Here is a screenshot from GitHub Action(I used wakatime-client back then but later I used axios to fix this 401 error as a workaround).

![gh](https://user-images.githubusercontent.com/33030965/89713239-f9736f80-d9c8-11ea-9390-0fd3dc000b1c.png)

Here is the [link]((https://github.com/tizee/waka-box/runs/913006736?check_suite_focus=true)) to my github workflow log.

Reply:
> I'd definitely prefer this value to match the value in `devDependencies`
Oooops,  it's a mistake😂.  The `axios` hasn't released its latest version(`0.20.0`) for fixing deep merging issues on configs. Currently it's `0.20.0-0`, which also stands for `next`(for now) so  I used it for sake of simplicity.

Reply:
No worries - totally understandable